processor/resourcedetectionprocessor/internal/gcp/metadata.yaml

```diff
@@ -48,4 +48,12 @@ resource_attributes:
   k8s.cluster.name:
     description: The k8s.cluster.name
     type: string
+    enabled: true
+  gcp.cloud_run.job.execution:
+    description: The Job execution name
+    type: string
+    enabled: true
+  gcp.cloud_run.job.task_index:
+    description: The Job execution task index
+    type: string
     enabled: true
```

Comment:
With the new metatada approach, we should not add attributes enabled by default at the first iteration as it's considered a breaking change. They should be optional initially. If we want to enable them by default, this should be done as a follow-up change in couple of versions and this PR should have a warning like this https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/metadata.yaml#L136-L137

Reply:
Hm... I've just found that this is applied only for CloudRunJob which was not covered before, so it's probably good to have them enabled right away

Reply:
@dmitryax is this good to resolve?